types/model/digest.go

```diff
@@ -47,8 +48,11 @@ var (
 // Digest.
 func ParseDigest(s string) Digest {
 	typ, digest, ok := strings.Cut(s, "-")
+	if !ok {
+		typ, digest, ok = strings.Cut(s, ":")
+	}
 	if ok && isValidDigestType(typ) && isValidHex(digest) {
-		return Digest{s: s}
+		return Digest{s: fmt.Sprintf("%s-%s", typ, digest)}
```

Comment:
- should only be used when referring to the filepath. : is still the canonical separator

Reply:
It would be nice to keep this consistent. Is there a real downside to do so? Having to convert back and forth mentally is a awkward and error prone.

Reply:
No real downsides other than backwards compatibility. If that's covered 🚢